e2e/testcases/resource_group_controller_test.go

```diff
@@ -123,28 +114,48 @@ func TestResourceGroupControllerInKptGroup(t *testing.T) {
 			},
 		},
 	}
-	nt.Must(testresourcegroup.UpdateResourceGroup(nt.KubeClient, rgNN, func(rg *v1alpha1.ResourceGroup) {
```

Comment:
FYI, UpdateResourceGroup works fine, but Get & Update is the same number of lines and more consistent with how other tests do the same thing.

We could probably remove UpdateResourceGroup, but it's used in a couple other files I didn't want to touch in this PR.